Resources/Private/Templates/NodeTypes/CheckBoxEnhanced.html

```diff
@@ -0,0 +1,25 @@
+{namespace neos=TYPO3\Neos\ViewHelpers}
+<f:layout name="{layoutName}" />
+<f:section name="Content">
+    <div class="checkbox {f:if(condition: validationResults.flattenedErrors, then: 'has-error')}">
+        <f:if condition="{neos:rendering.inBackend()}">
+            <f:then>
+                <f:if condition="{mainText}">
+                    {mainText -> f:format.raw()}
+                </f:if><br/><br/>
```

Comment:
Do not hardcode styling in HTML ("br+br"), instead put things in containers (and potentially with classes, if needed) and use CSS for stying / spacing (in this case, as part of DAV-Shop base project's CSS - if required).